src/neuroconv/tools/nwb_helpers/_dataset_configuration.py

```diff
@@ -100,17 +103,25 @@ def get_default_dataset_io_configurations(
 
     for neurodata_object in nwbfile.objects.values():
         if isinstance(neurodata_object, DynamicTable):
-            dynamic_table = neurodata_object  # for readability
+            dynamic_table = neurodata_object  # For readability
 
             for column in dynamic_table.columns:
                 candidate_dataset = column.data  # VectorData object
                 if _is_dataset_written_to_file(
                     candidate_dataset=candidate_dataset, backend=backend, existing_file=existing_file
                 ):
-                    continue  # skip
+                    continue  # Skip
 
                 # Skip over columns that are already wrapped in DataIO
                 if isinstance(candidate_dataset, DataIO):
+                    continue  # Skip
+
+                # Skip over columns whose values are links, such as the 'group' of an ElectrodesTable
+                if any(isinstance(value, Container) for value in candidate_dataset):
+                    continue  # Skip
+
+                # Skip when columns whose values are a reference type
+                if isinstance(column, TimeSeriesReferenceVectorData):
```

Comment:
This was an nteresting case caught by the now pretty extensive mixin testing - particular to the ABF interface over in `icephys` land